docs/notebooks/basic-scrna-tutorial.ipynb

```diff
@@ -0,0 +1,1917 @@
+{
```

Comment:
<p>Careful! You are linking to the surface protein aka CITE-seq markers here. You might need to clarify this.</p><p> </p><p>The actual RNA annotation chapter PR is here: https://github.com/theislab/single-cell-best-practices/pull/85 it is like 95% done and should hopefully get merged soon.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>